app/src/pages/admin/pluginsPage/pluginsListItems/pluginsItem/pluginsItem.jsx

```diff
@@ -132,7 +134,10 @@ export class PluginsItem extends Component {
         <div className={cx('plugins-additional-block')}>
           {toggleable && (
             <div className={cx('plugins-switcher')} onClick={(e) => e.stopPropagation()}>
-              <InputSwitcher value={this.state.isEnabled} onChange={this.onToggleActiveHandler} />
+              <InputSwitcher
```

Comment:
According to the task description: `Disable/enable switcher should be removed form the main plugin tab`.

Reply:
Please, see the comment https://jiraeu.epam.com/browse/EPMRPP-48945?focusedCommentId=6695048&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6695048